doc/callbacks.md

```diff
@@ -84,12 +84,16 @@ has been made.
 
  * `EarlyStopAtMinLoss()`
  * `EarlyStopAtMinLoss(`_`patience`_`)`
+ * `EarlyStopAtMinLoss(`_`func`_`)`
+ * `EarlyStopAtMinLoss(`_`func`_`,`_`patience`_`)`
 
 #### Attributes
 
 | **type** | **name** | **description** | **default** |
 |----------|----------|-----------------|-------------|
 | `size_t` | **`patience`** | The number of epochs to wait after the minimum loss has been reached. | `10` |
+| `std::function<double(const MatType&)>` | **`func`** | A callback to return immediate loss evaluated by the function. | |
+
```

Comment:
```suggestion
| `std::function<double(const arma::mat&)>` | **`func`** | A callback to return immediate loss evaluated by the function. | |

Note that for the `func` argument above, if a
[different matrix type](#alternate-matrix-types) is desired, instead of using
the class `EarlyStopAtMinLoss`, the class `EarlyStopAtMinLossType<MatType>`
should be used.
```

(This goes with [this comment](https://github.com/mlpack/ensmallen/pull/165/files#r450571058).)

If you want to rephrase my comment go ahead---this is just a suggestion. :+1: